structurizr-plantuml/src/com/structurizr/io/plantuml/PlantUMLWriter.java

```diff
@@ -27,10 +27,10 @@
 public class PlantUMLWriter {
 
     /** maximum diagram width or height; defaults to 2000px to match public plantuml.com installation */
-    private int sizeLimit = 2000;
```

Comment:
Happy to change the access modifiers of the methods, but I think I'd prefer to keep the properties as private.